docker-for-mac/index.md

```diff
@@ -131,6 +133,16 @@ You can set the following runtime options.
   Uncheck this to opt out and prevent auto-send of data. Docker may prompt for
   more information in some cases, even with auto-send enabled.
 
+#### Choose whether to store Docker credentials on Mac keychain (Edge feature)
```

Comment:
I also have some Edge feature notes that provide links and give more info that I don't want to replace. I'm testing this to see what it gives me.